cmd/rollback/main.go

```diff
@@ -35,11 +37,25 @@ func main() {
 		zap.S().Fatalf("Initialization error: %v", err)
 	}
 
-	cfg, err := settings.BlockchainSettingsByTypeName(*blockchainType)
-	if err != nil {
-		zap.S().Error(err)
-		return
+	var cfg *settings.BlockchainSettings
+	if *cfgPath != "" {
```

Comment:
It is not a problem at all. Configuration overrides blockchain type selection. Actually providing "mainnet" or "testnet" to `-blockchain-type` lead to loading of configuration from corresponding embedded file.

Reply:
> Actually providing "mainnet" or "testnet" to -blockchain-type lead to loading of configuration from corresponding embedded file.

But it seems in this case mainnet configuration wouldn't be loaded. And if in future we add other usage of `blockchainType` (for example in logs) it will display non-actual blockchain type.